roles/router/tasks/uninstall_sep_gnb_nw.yml

```
@@ -0,0 +1,118 @@
+---
+
+- set_fact:
+    systemd_network_dir: /etc/systemd/network
+
+- name: "remove a static route on srs host for {{ core.upf.ip_prefix }} via {{ core.amf.ip }}"
+  shell: |
+    ip route del {{ core.upf.ip_prefix }} via {{ core.amf.ip }}
+  when: (inventory_hostname in groups['srs_nodes'][0]) and (inventory_hostname not in groups['master_nodes'])
+  become: true
+  ignore_errors: yes
+
+- name: "remove a static route on core host for {{ srs.docker.network.subnet }} via {{ hostvars[item]['ansible_default_ipv4']['address'] }}"
```

Should not this be changed to `srsran` across the file? Or what is this file used for?
```suggestion
- name: "remove a static route on core host for {{ srsran.docker.network.subnet }} via {{ hostvars[item]['ansible_default_ipv4']['address'] }}"
```

---

file removed